engine/table/src/main/java/io/deephaven/engine/table/impl/BlinkTableTools.java

```diff
@@ -26,21 +24,55 @@
  * @see Table#BLINK_TABLE_ATTRIBUTE
  */
 public class BlinkTableTools {
+    private static final Object DEFAULT_MEMO_KEY = new Object() {
+        @Override
+        public String toString() {
+            return "DEFAULT_MEMOIZATION_KEY";
+        }
+    };
 
     /**
      * Convert a Blink Table to an in-memory append only table.
      * <p>
-     * Note, this table will grow without bound as new blink table rows are encountered.
+     * Note, this table will grow without bound as new blink table rows are encountered. The result is memoized under
+     * {@link #DEFAULT_MEMO_KEY}.
```

Comment:
If we're going to link to it, it _should_ be public IMO. I think it's _ok_ to have it public.